src/Microsoft.AspNetCore.Diagnostics.HealthChecks/Microsoft.AspNetCore.Diagnostics.HealthChecks.csproj

```diff
@@ -18,6 +18,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="$(MicrosoftAspNetCoreHttpAbstractionsPackageVersion)" />
     <PackageReference Include="Microsoft.Extensions.Options" Version="$(MicrosoftExtensionsOptionsPackageVersion)" />
+    <PackageReference Include="Microsoft.Net.Http.Headers" Version="$(MicrosoftNetHttpHeadersPackageVersion)" />
```

Comment:
Why do you need this?

Reply:
`HeaderNames` is defined there for some reason - is there a problem with using it?

Reply:
Wow I had no idea we put it there. In general, I'd like this package to die a slow death. We need to re-unify with the real typed headers in .NET Core. Anyways, it's fine to have this for now I guess.

Reply:
Yeah 😞 I spent a while looking for HeaderNames because I thought it surely couldn't be defined in *that* package.